carbon-p2-plugin/src/main/java/org/wso2/maven/p2/beans/CarbonArtifact.java

```diff
@@ -0,0 +1,113 @@
+/*
+ *  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License format used is obsolete. Please update IDE's copyright profiles with the appropriate format.